recipe/meta.yaml

```diff
@@ -45,7 +45,7 @@ requirements:
     - xlrd >=1.0
     - h5py >=2.4
     - setuptools >=1.0
-    - pyqt 4.11.*
+    - pyqt
```

Comment:
Same comment as https://github.com/conda-forge/glue-vispy-viewers-feedstock/pull/17/files#r107933779

With one extra. Note that conda can "wiggle" its way into `qt 4` or at `qt 5` here depending on the other dependencies pinning and the number/order of other channels . So pinning to `pyqt 5.6.*` is probably safer.